README.md

```diff
@@ -109,10 +109,26 @@ The easiest way to install LocalStack is via `pip`:
 pip install localstack
 ```
 
+Installing the `awscli` for running the aws commands through `cli`
+
+```
+pip install awscli
+```
+
 **Note**: Please do **not** use `sudo` or the `root` user - LocalStack
 should be installed and started entirely under a local non-root user. If you have problems
 with permissions in MacOS X Sierra, install with `pip install --user localstack`
 
+## Setting up local region and credentials to run localstack
```

Comment:
Can we please move this paragraph into the `Accessing the infrastructure via CLI or code` section further below.